integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMultiDomainModels.java

```diff
@@ -130,7 +130,6 @@ void testScaleClustersAndAdminConsoleLogin(String domainType) {
     // In OKD cluster, we need to get the routeHost for the external admin service
     String routeHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), domainNamespace);
 
-
```

Comment:
Do we not verify these usecase scale and access the console on three type of domains in other classes.  If so why we need this class

Reply:
If this is for gating test, can we add a Auxiliary Image model.

Reply:
done